### DISCUSSION TITLE
HypergraphConv hyperedge_attr

### DISCUSSION THREAD

**Discussion by 1zzt** [pyg-team/pytorch_geometric#6976] [Q&A]
I want to modify HypergraphConv class to make hyperedge_attr can be changed and saved after a certain message. What should I do?

**Comment by rusty1s** [comment]
You can modify `HyperGraphConv` return `hyperedge_attr` as part of its `forward` method.

**Reply by 1zzt** [reply]
Thank you ! 
For a pairwise graph, I want to initialize the edge_attr, what is the order of the bond_features? Is it in the order of edge_index?

**Reply by rusty1s** [reply]
`hyperedge_attr` refers to a feature for each hyperedge/each cluster in your graph, so it's shape should be `[M, F]`, where `M` refers to the number of clusters, e.g., `M = hyperedge_index[1].max() + 1`.
